### PR TITLE
Assignments 2,3,4

### PR DESCRIPTION
**Pull request by AFGranado**
Corrección de assignment 2 y entrega de 3 y 4.

### CODE REVIEW

File: Assignment2/AFGranado-AnlleloGranado.ttl [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+		vocab:includes :Computer101;
+
+:Sensor029	vocab:hasMeasurement :Measurement8401;
+
+:Measurement8401	vocab:hasTemperature "29"^^xsd:int;
+			vocab:atTime "2010-06-12T12:00:12"^^xsd:dateTime.
+
+:Computer101	vocab:hasOwner :User10A.
+
+:User10A	vocab:hasName :"Pedro"^^xsd:string.
```
Comment (ocorcho) [inline Assignment2/AFGranado-AnlleloGranado.ttl]: This is still wrong. No : symbol before the string "Pedro", please.

**Comment by ocorcho** [comment]
Assignment 3 is ok. Assignment 2 needs corrections. And @dachafra will check assignment 4

File: Assignment4/AFGranado-w140276/Task06.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+		// ** TASK 6.4: Create a new individual of Researcher named "Jane Smith" **
+		Individual janeSmith = researcher.createIndividual(ns+"Jane Smith");
+		
+		// ** TASK 6.5: Add to the individual JaneSmith the fullName, given and family names **
+		janeSmith.addLiteral(VCARD.FN, "Jane Smith");
+		janeSmith.addLiteral(VCARD.Given, "Jane");
+		janeSmith.addLiteral(VCARD.Family, "Smith");
+		
+		// ** TASK 6.6: Add UPM as the university where John Smith works **
+		Individual UPM = university.createIndividual(ns+"UPM");
```
Comment (dachafra) [inline Assignment4/AFGranado-w140276/Task06.java]: the task does not say anything about creating the individuals

**Comment by dachafra** [comment]
task 6 needs some improvements

**Comment by dachafra** [comment]
waiting for the changes in assign2 to accept the PR

**Comment by AFGranado** [comment]
> 
> 
> waiting for the changes in assign2 to accept the PR
I corrected assignment 2 7 days ago. Is it still wrong?

File: Assignment2/AFGranado-AnlleloGranado.ttl (hunk 1 above)
Comment (dachafra) [reply Assignment2/AFGranado-AnlleloGranado.ttl]: reviewed and it is correct now

**Comment by dachafra** [comment]
Well done!

**Pull request merged**